examples/apps/aml/web-client/src/App.vue

```diff
@@ -0,0 +1,259 @@
+<!-- 
+  Copyright 2019 Banco Santander S.A.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. 
+-->
+
+<template>
+  <div id="app">
+    <b-container>
+      <div class="text-center m-4">
+        <b-img :src="logo" fluid alt="Santander Logo"></b-img>
+      </div>
+
+      <h3 class="mt-3">Available workers</h3>
+
+      <div class="text-center" v-if="workerList === null">
+        <h4>Looking up Workers</h4>
+        <b-spinner variant="danger" label="Loading..."></b-spinner>
+      </div>
+      <div v-else>
+        <b-table :items="workerList" :fields="fields" class="mt-3" outlined>
+          <template slot="show_details" slot-scope="row">
+            <b-button
+              size="sm"
+              @click="requestDelete(row.item.Worker_ID_public_EC_key)"
+              class="mr-2"
+              variant="outline-primary"
+            >Details</b-button>
+          </template>
+        </b-table>
+      </div>
+
+      <b-modal size="lg" title="Worker Details" ref="modal" @show="nullWorker" ok-only>
```

Comment:
This component seems to break on chrome 78.0.39. 
![image](https://user-images.githubusercontent.com/37328891/69845947-6fe79b80-1298-11ea-8d46-58eed632fd50.png)